radis/lbl/calc.py

```diff
@@ -129,6 +132,17 @@ def calc_spectrum(
 
             mole_fraction={'CO2': 0.8, 'CO':0.2}​
 
+    diluent: str or dictionary
+            can be a string of a single diluent or a dictionary containing diluent
+            name as key and its mole_fraction as value
+            For single diluent ::
+
+            diluent = 'CO2'
```

Comment:
```suggestion
                diluent = 'CO2'
```
I think you need an offset so it appears as code